docs/source/tutorials/datalab.ipynb

```diff
@@ -0,0 +1,1393 @@
+      "127              True       0.076251                76                  2.297110e+00\n",
+      "128              True       0.083941                57                  2.199371e+00\n"
+     ]
+    }
+   ],
+   "source": [
+    "lab.report()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. View issue summaries and additional information regarding the dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are several methods to get more details about a particular issue.\n",
+    "\n",
+    "The `get_summary()` method fetches summary statistics regarding how severe each type of issue is overall across the whole dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>issue_type</th>\n",
+       "      <th>score</th>\n",
+       "      <th>num_issues</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>label</td>\n",
+       "      <td>0.909091</td>\n",
+       "      <td>11</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>outlier</td>\n",
+       "      <td>0.522080</td>\n",
+       "      <td>6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>near_duplicate</td>\n",
+       "      <td>0.246459</td>\n",
+       "      <td>8</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       issue_type     score  num_issues\n",
+       "0           label  0.909091          11\n",
+       "1         outlier  0.522080           6\n",
+       "2  near_duplicate  0.246459           8"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lab.get_summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also pass the type of issue as a argument to the `get_summary()` method to get the summary for a particular issue. For example, we can pass in `label` as an argument to only get the summary regarding label issues."
```

Comment:
```suggestion
    "We can also only request the summary for a particular type of issue."
```

Reply:
the text in english should generally not be a verbatim explanation of every little detail in the code, when these are obvious